.flowconfig

```diff
@@ -23,7 +23,11 @@
 .*/Libraries/polyfills/.*
 
 ; Ignore metro
-; .*/node_modules/metro/.*
+.*/node_modules/metro/.*
```

Comment:
Is sdx-platform locked for RN changes yet? If not, we could get into a lot of painful merge conflicts next time we merge this fork into it

Reply:
You need to bypass various checks to get it into sdx-platform these days.  We should be good here.